docs/source/concept_models/vivarium_us_cvd/.smbdeleteAAA164eb10d

```diff
@@ -0,0 +1,338 @@
+.. _us_cvd_concept_model:
```

Comment:
I assume you don't want this file in your PR. You'll need to remove it (`git rm`) locally, commit, and push to do that. Or I think there's also a way to delete this file in the Github web UI.